### DISCUSSION TITLE
rtu

### DISCUSSION THREAD

**Discussion by domgggg** [MikaylaFischler/cc-mek-scada#435] [Support] [answered]
On the rtu screen on "network" the light to the left is yellow and not green but it has a wired modem and a wireless one too.

**Comment by MikaylaFischler** [comment, marked as answer]
For the RTU per [here](https://github.com/MikaylaFischler/cc-mek-scada/wiki/RTU-Gateway):

NETWORK: gray for no connection acknowledged yet, red if supervisor connection denied generically, orange for version mismatch, green for successful link with supervisor.

Note that since there was a release last week, if you installed or updated some devices since then, they won't talk to devices that weren't up-to-date since the communications protocol version changed

**Reply by domgggg** [reply]
okay thanks
